platform/config/__tests__/ensureDeepObject.test.ts

```diff
@@ -4,7 +4,7 @@ test('flat object', () => {
   const obj = {
     'foo.a': 1,
     'foo.b': 2
-  }
+  };
```

Comment:
I just added semi-colons where my IDE wasn't happy enough :) But do you think we should re-think where we put them and where we don't?